apps/cnspec/cmd/scan.go

```diff
@@ -220,6 +221,14 @@ func getCobraScanConfig(cmd *cobra.Command, runtime *providers.Runtime, cliRes *
 		optAnnotations[k] = v
 	}
 
+	assetName, err := cmd.Flags().GetString("asset-name")
+	if err != nil {
+		log.Fatal().Err(err).Msg("failed to parse asset-name")
+	}
+	if assetName != "" {
+		cliRes.Asset.Name = assetName
```

Comment:
is it guaranteed that the `cliAsset` is not nil?

Reply:
Not sure, I'll add a check